contracts/deploy-ready/ERC721MinterPauser.sol

```diff
@@ -21,34 +22,45 @@ import "../token/ERC721/ERC721Pausable.sol";
  * both the minter and pauser roles.
  */
 contract ERC721MinterPauser is Context, AccessControl, ERC721Burnable, ERC721Pausable {
```

Comment:
The addition of auto incremented token ids makes this name no longer accurate, to be honest. I preferred the idea of "deploy ready" where we define whatever that means for each contract.

Reply:
`ERC721OutOfTheBox`
`ERC721ReadyToUse`
`ERC721Standard`
`ERC721Essentials`

Reply:
`ERC721DeployReady`

Reply:
ERC721Preset

Reply:
I'd rather keep the name descriptive, we might have in the future multiple variants for each token type. And a non-descriptive name forces you to be on top of the docs to know what those are, I'd rather keep using them simple.

Regarding this one in particular, I'd consider dropping the Pausable bit from it, which is quite unrelated to the other features, and leave it as a mintable 721 with autogenerated IDs and URIs.

Reply:
I’d like Pausable dropped. Minting with automatic token URIs should be well used. I don’t think there will be much need for pause.